packages/contracts/test/core/plugin/shared-plugin.ts

```diff
@@ -160,7 +160,7 @@ describe('SharedPlugin', function () {
       await ethers.provider.send('evm_mine', []);
 
       // The call is allowed for the allowed ID
-      await expect(testPlugin.callStatic.idGatedAction(allowedId)).to.not.be
+      await expect(testPlugin.callStatic.idGatedAction(allowedId)).not.to.be
```

Comment:
Do these particles have an impact?

Reply:
Nope, but still made it for consistency..

Reply:
The wrong thing was another fix(quite important) (`await` must be outside `expect` instead of inside)